lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -357,24 +365,26 @@ def decrement_positions_on_lower_items(position=nil)
           end
 
           # This has the effect of moving all the higher items down one.
-          def increment_positions_on_higher_items
+          def increment_positions_on_higher_items(avoid_id = nil)
             return unless in_list?
             acts_as_list_class.unscoped do
               acts_as_list_class.where(scope_condition).where(
-                "#{position_column} < #{send(position_column).to_i}"
+                "#{position_column} < #{send(position_column).to_i} AND #{avoid_id_condition(avoid_id)}"
               ).update_all(
                 "#{position_column} = (#{position_column} + 1)"
               )
             end
           end
 
+          def avoid_id_condition(avoid_id)
+            avoid_id ? "#{self.class.primary_key} != #{self.class.connection.quote(avoid_id)}" : '1 + 1'
```

Comment:
Line is too long. [104/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.